tools/BlobRewriter/BlobRewriter.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <AssemblyName>Microsoft.Health.Internal.Fhir.BlobRewriter</AssemblyName>
+    <AssemblyName>Fhir.BlobRewriter</AssemblyName>
```

Comment:
May we keep the original assembly name?

Reply:
It is very inconvenient to have long exe name with all not distinct letters in front. This way I don't need to type much.